experimental/traffic-portal/src/app/api/profile.service.ts

```diff
@@ -103,4 +124,64 @@ export class ProfileService extends APIService {
 		return this.delete<ResponseProfile>(`profiles/${id}`).toPromise();
 	}
 
+	public async getParameters(idOrName: number | string): Promise<ResponseParameter>;
+	public async getParameters(): Promise<Array<ResponseParameter>>;
+	/**
+	 * Retrieves Parameters from the API.
+	 *
+	 * @param idOrName Specify either the integral, unique identifier (number) of a specific Parameter to retrieve,
+	 * or its name (string).
+	 * @returns The requested Parameter(s).
+	 */
+	public async getParameters(idOrName?: number | string): Promise<Array<ResponseParameter> | ResponseParameter> {
```

Comment:
Parameters cannot be uniquely identified by name. Any number of Parameters can - and do, for the vast majority of Parameters - exist by the same name at any given time. So if you want to be able to filter Parameters by name that's fine, but neither this method itself nor the caller should expect that doing so will return a single Parameter.